department-of-veterans-affairs/caseflow
# Issue 8924 - Remove hearing task from RootTask available actions.

Resolves #8924 

### Description
Removed Schedule Hearing from list of available actions at the RootTask level.

### Acceptance Criteria

### Testing Plan
1. Go to CaseDetails
2. Confirm Schedule Hearing task is not in actions dropdown options
